main.go

```diff
@@ -121,21 +117,29 @@ func main() {
 		Flags:  flags,
 		Before: altsrc.InitInputSourceWithContext(flags, altsrc.NewTomlSourceFromFlagFunc(CONFIG)),
 		Action: func(cCtx *cli.Context) error {
+			if pkString == "" {
```

Comment:
I think if we set `Required: true` when constructing the flag then `urfave` will handle making sure that a value is provided. (See `naaddress`)

Reply:
https://github.com/urfave/cli/issues/849